packages/frontend/src/components/ProjectConnection/WarehouseForms/DatabricksForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC } from 'react';
+import { Button } from '@blueprintjs/core';
+import { useToggle } from 'react-use';
+import Input from '../../ReactHookForm/Input';
+import NumericInput from '../../ReactHookForm/NumericInput';
+import PasswordInput from '../../ReactHookForm/PasswordInput';
+
+const DatabricksForm: FC<{
+    disabled: boolean;
+}> = ({ disabled }) => {
+    const [isOpen, toggleOpen] = useToggle(false);
+    return (
+        <>
+            <Input
+                name="warehouse.serverHostName"
+                label="Server host name"
+                documentationUrl="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#server-host-name"
+                rules={{
+                    required: 'Required field',
+                }}
+                disabled={disabled}
+            />
+            <Input
+                name="warehouse.httpPath"
+                label="HTTP Path"
+                documentationUrl="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#http-path"
+                rules={{
+                    required: 'Required field',
+                }}
+                disabled={disabled}
+            />
+            <PasswordInput
+                name="warehouse.personalAccessToken"
+                label="Personal access token"
+                documentationUrl="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#personal-access-token"
+                rules={{
+                    required: 'Required field',
+                }}
+                disabled={disabled}
+            />
+            <Input
+                name="warehouse.database"
+                label="database"
+                documentationUrl="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#database"
```

Comment:
This url and the ones bellow need to be updated to point to the databricks section. There are multiple titles with the name `database` and each has a different url E.g `database-1` `database-2`